modules/SerialCommHub/manifest.yaml

```diff
@@ -24,6 +24,10 @@ provides:
         description: On some hardware every message that is sent is read back, this setting filters the sent message in the reply.
         type: boolean
         default: false
+      skip_zero_padding:
```

Comment:
This should probably not be a config option for the module. The problem is that it applies to all devices on the bus if you set it here, it probably is only needed for one of them? In that case it may be better to implement this as an argument for the read command (with a default disabled value) so that multiple devices can share the same bus. Also we should make sure this is really needed and is not a result of e.g. some misconfiguration of the serial port settings (in which case it would be work around code for hiding a bug)

Reply:
This is actually the reason for this PR: 
A lot of times, when debugging new hardware, there is no clean bus, in which case one can set this option to remove timing issues introduced by non-clean bus connections. In this case it affects ALL communication on the bus and since the SerialCommHub is the module responsible for controlling the bus, I think this would be the best place to be able to set this option. 
Since I have run into this problem more than once, I think this option might add some value for other developers working on integrating new devices as well.
It has a default of "false", thus need not be considered in any situations that do not expressly need this option, but for those which need it there is no good other way...

Reply:
Do we know where these extra bytes are coming from? Are we sure they are always zero?

Reply:
Yes, they are always zero. They seem to come from bus noise/too wide timing windows/too large capacitance on the bus. I would like to have measured with a scope but Julian is with you at the camp right now...